wp-modules/api-data/tests/ApiDataTest.php

```diff
@@ -1,98 +0,0 @@
-<?php
-/**
- * Class ApiDataTest
- *
- * @package pattern-manager
- */
-
-// phpcs:disable WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound
-
-namespace PatternManager\ApiData;
-
-use WP_UnitTestCase;
-use WP_REST_Request;
-
-require_once dirname( __DIR__ ) . '/api-data.php';
-
-/**
- * Test the api functions.
- */
-class ApiDataTest extends WP_UnitTestCase {
-
-	/**
-	 * @inheritDoc
-	 */
-	public function setUp() : void {
-		parent::setUp();
-		add_filter( 'request_filesystem_credentials', '__return_true' );
-		add_filter( 'stylesheet_directory', [ $this, 'get_fixtures_directory' ] );
-	}
-
-	/**
-	 * @inheritDoc
-	 */
-	public function tearDown() : void {
-		unset( $GLOBALS['wp_rest_server'] );
-		remove_filter( 'request_filesystem_credentials', '__return_true' );
-		remove_filter( 'stylesheet_directory', [ $this, 'get_fixtures_directory' ] );
-		parent::tearDown();
-	}
-
-	/**
-	 * Gets the fixtures directory.
-	 */
-	public function get_fixtures_directory() {
-		return dirname( dirname( __DIR__ ) ) . '/pattern-data-handlers/tests/fixtures';
-	}
-
-	/**
-	 * Tests register_routes.
-	 */
-	public function test_register_routes_get_pattern_names_unauthorized() {
-		wp_set_current_user( $this->factory()->user->create( [ 'role' => 'author' ] ) );
-		do_action( 'rest_api_init' );
-
-		$this->assertSame(
-			'rest_forbidden',
-			rest_do_request( new WP_REST_Request( 'GET', '/pattern-manager/v1/get-pattern-names' ) )->get_data()['code']
```

Comment:
I wrote most of this file 😄 

But it's really testing the WP API, not this plugin.

For example, that an author can't access the API endpoint.